src/servers/src/http.rs

```diff
@@ -105,7 +107,7 @@ pub(crate) async fn query_context_from_db(
 pub const HTTP_API_VERSION: &str = "v1";
 pub const HTTP_API_PREFIX: &str = "/v1/";
 /// Default http body limit (64M).
-const DEFAULT_BODY_LIMIT: usize = 64 * 1024 * 1024;
+const DEFAULT_BODY_LIMIT: &str = "64M";
```

Comment:
suggestion: `ReadableSize::mb()` is a constant function, so you could try
```suggestion
const DEFAULT_BODY_LIMIT: ReadableSize = ReadableSize::mb(64);
```

Refer to
https://github.com/GreptimeTeam/greptimedb/blob/884731a2c807fe5d0e5ac86bb5863d4a06bfeafd/src/common/base/src/readable_size.rs#L45